chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -eou pipefail
+
+readonly BACKUP_MANIFEST_URL='https://gist.githubusercontent.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/030a82af2489bc353cec370779cfd06852668e29/velero.yaml'
+readonly KUBE_NAMESPACES_URL='https://gist.githubusercontent.com/dmitry-mightydevops/297c4e235b61982f21a0bbbf7319ac24/raw/0e57e62984b69ff16c9b9159776e145f6d4feecf/kubernetes-namespaces.txt'
+
+
+#########################################
+# Get backup manifest 
+# Arguments:
+#   Backup mainfest
+# Outputs:
+#   Included namespaces
+#########################################
+get_included_namespaces_from_backup() {
+  local backup="${1}"
+  echo "${backup}"| yq ".spec.source.helm.values" -r | yq ".schedules[].template.includedNamespaces[]" -r
+}
+
+###############################################
+# Fetch data.
+# Arguments:
+#  URL 
+# Outputs:
+#   Fetch result or or terminates the script 
+###############################################
+fetch_data() {
+  local url="$1"
+  local response_status=$(curl -I --silent "${url}" | head -n 1 | cut -d' ' -f2)
```

Comment:
```suggestion
  local response_status
  response_status=$(curl -I --silent "${url}" | head -n 1 | cut -d' ' -f2)
```

https://unix.stackexchange.com/a/687856